components/esp_modem/examples/modem_console/main/modem_console_main.cpp

```diff
@@ -161,6 +215,7 @@ extern "C" void app_main(void)
                 dev_mode = esp_modem::modem_mode::DATA_MODE;
             } else if (mode == "CMUX") {
                 dev_mode = esp_modem::modem_mode::CMUX_MODE;
+                mode_rtc = dev_mode;
```

Comment:
Please add a comment, that even if switching to `CMUX` fails, we make the DTE multiplex terminal. (this is potentially a bug and might be fixed eventually)

Reply:
Done